src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -553,6 +567,13 @@ public static DynamoModel Start(IStartConfiguration configuration)
         /// <param name="config">Start configuration</param>
         protected DynamoModel(IStartConfiguration config)
         {
+            if (config is DefaultStartConfiguration)
+            {
+                // This is not exposed in IStartConfiguration to avoid a breaking change.
+                // This fact should probably be revisited in next major.
+                defaultPythonEngine = ((DefaultStartConfiguration)config).DefaultPythonEngine;
```

Comment:
would it ever happen that we've loaded an old xml config and this will throw?

Reply:
I don't know in which context this gets serialized/deserialized. XML serialization OOTB does not throw in cases like this but I don't know what you are doing exactly. Can you point to where that happens?